pkg/xray/resource/resource_xray_repository_config.go

```
@@ -55,47 +55,49 @@ func (m RepoConfigResourceModel) toAPIModel(_ context.Context, xrayVersion, pack
 		var vulnContextualAnalysis *bool
 		var exposures *ExposuresAPIModel
 
-		if m.JASEnabled.ValueBool() {
-			if slices.Contains(vulnContextualAnalysisPackageTypes(xrayVersion), packageType) {
-				vulnContextualAnalysis = configAttrs["vuln_contextual_analysis"].(types.Bool).ValueBoolPointer()
-			}
+		// The Xray REST API requires `vuln_contextual_analysis` and `exposures` in the
+		// payload regardless of the JAS license state (jas_enabled). Omitting them causes
+		// the server to return: "Request payload is invalid as exposure analysis config is missing".
+		// See GH issue: xray_repository_config - JAS configuration required for non-JAS environments.
+		if slices.Contains(vulnContextualAnalysisPackageTypes(xrayVersion), packageType) {
```

golangci-lint (2.12.2)

[error] inline: cannot inline: type parameter inference is not yet supported

(govet)